zabbix/assets/configuration/spec.yaml

```
@@ -0,0 +1,10 @@
+name: zabbix
+files:
+- name: zabbix.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

```suggestion
    - template: instances/http
    - template: instances/default
```

If this integration uses the HTTP wrapper (wrapper around the Requests library), this config template will automatically support other requests options like (tls verify, ca cert, etc).

---

I changed the code to use HTTP wrapper, and I added `- template: instances/http` as suggested.